backoffice/src/Services/MigrationService.cs

```diff
@@ -257,5 +259,26 @@ public async Task<WreckMaterial> MigrateWreckMaterialAsync(PowerappsWreckMateria
 
         return wreckMaterial;
     }
+
+
+    public async Task HandleTriageCsvAsync(List<TriageRowDto> records)
+    {
+        // loop through records
+        foreach ( var record in records )
+        {
+            try
+            {
+                var droit = await _droitService.GetDroitByReferenceAsync(record.DroitReference);         
+                droit.TriageNumber = int.Parse(record.TriageNumber);
```

Comment:
What happens if there's an empty triage cell?, might be worth having some extra error handling around the data provided

Reply:
still need to do the try, and also some sort of validation it's between 1-5